scripts/download_srs_aztec.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Check if the .env file exists
+if [ -f .env ]; then
+    # Load variables from .env into the environment
+    source .env
+    echo "Environment variables loaded from .env"
+else
+    echo ".env file not found"
+fi
+
+if [ -f "$AZTEC_SRS_PATH" ]; then
+    echo "SRS file $AZTEC_SRS_PATH exists"
+else
+    echo "SRS file $AZTEC_SRS_PATH does not exist, downloading ..."
+    gh release download --repo alxiong/ark-srs v0.2.0 -p "$(basename $AZTEC_SRS_PATH)" -O "$AZTEC_SRS_PATH" --skip-existing
```

Comment:
Is it wise to use the `gh` tool when not everyone will have it installed? Also, should we pehaps pin this download into a company-owned github repository instead of a personal one?

Reply:
switch to `wget` and removed `gh` in the `buildInputs` in [47870cb](https://github.com/EspressoSystems/HotShot/pull/2854/commits/47870cb78cbbc2fb81f615f134b1140201ffbc29)